python/lsst/source/injection/inject_visit.py

```diff
@@ -128,3 +144,50 @@ def runQuantum(self, butler_quantum_context, input_refs, output_refs):
         input_keys = ["injection_catalogs", "input_exposure", "sky_map", "psf", "photo_calib", "wcs"]
         outputs = self.run(**{key: value for (key, value) in inputs.items() if key in input_keys})
         butler_quantum_context.put(outputs, output_refs)
+
+    """
+    Establish the variance scale by a linear fit of variance vs. flux.
+    In practice we see that most pixels in a coadd obey a consistent, simple
+    linear relationship between variance and flux, but a small sample skews
+    somewhat away from linearity and results in a fit that seems less than
+    ideal.
+
+    To identify and hence ignore these odd pixels, we perform a RANSAC fit.
+    RANSAC iteratively finds a least-squares straight line fit on random
+    subsamples of points, while identifying outliers. The final results tend to
+    be much more stable and outlier-robust than a simple linear fit.
+
+    Pixels below a flux level specified by brightness_cutoff are more likely to
+    have wild outliers, so for simplicity's sake we simply ignore them.
+    """
```

Comment:
Perhaps this should also be moved into a doc-string for the `get_variance_scale` method.